code/modules/food_and_drinks/recipes/recipes_oven.dm

```diff
@@ -293,6 +293,11 @@
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/sliceable/bread
 
+/datum/recipe/oven/bread/make_food(obj/container)
+	var/obj/item/weapon/reagent_containers/food/snacks/sliceable/bread/being_cooked = ..(container)
```

Comment:
No need to call container here, ..() automatically includes all of the procs parameters.